citation/models.py

```diff
@@ -697,6 +708,31 @@ def __str__(self):
                                                              container=self.container)
 
 
+class URLStatusLogs(models.Model):
+    PLATFORM_TYPES = Choices((PlatformTypes.COMSES, _('CoMSES')),
+                     (PlatformTypes.OPEN_SOURCE, _('Open Source')),
+                     (PlatformTypes.PLATFORM, _('Platform')),
+                     (PlatformTypes.JOURNAL, _('Journal')),
+                     (PlatformTypes.PERSONAL, _('Personal')),
+                     (PlatformTypes.INVALID, _('Invalid')),
+                     (PlatformTypes.OTHERS, _('Others'))
+                     )
+    pub_id = models.ForeignKey(Publication, related_name='url_status', null=True, blank=True, db_constraint=False)
+    url = models.URLField(blank=True, max_length=500)
+    date_added = models.DateTimeField(auto_now_add=True,
+                                      help_text=_('Date this url was last verified'))
+    date_modified = models.DateTimeField(auto_now=True,
```

Comment:
our codebase convention is to name these fields `date_created` and `last_modified` - please stick with conventions so we don't need to access these types of fields using different names for different models.